mixcoatl/resource.py

```diff
@@ -8,7 +8,89 @@
 from mixcoatl.settings.load_settings import settings
 from mixcoatl.decorators.lazy import lazy_property
 from mixcoatl.utils import camel_keys
+import json
 
+class Endpoint(object):
+    """ A class for representing DCM endpoints and loading them from json files
+    """
+
+    def __init__(self, url = None, basepath=None, api_version=None, secret_key=None, access_key=None, ssl_verify=True, nickname=None):
```

Comment:
On the contrary, I'm thrilled that I'm not the only one who is going to be making PEP8 related comments. Hit me with these anytime. A PEP8 commit is pending shortly.